drivers/usb/device/Kconfig

```diff
@@ -50,4 +50,37 @@ config SYS_LOG_USB_DRIVER_LEVEL
 
 	- 4 DEBUG, write SYS_LOG_DBG in addition to previous levels
 
+config USB_DC_STM32_DISCONN_ENABLE
```

Comment:
why all this stuff is configurable by 'user'

Reply:
You are right, this should should be configured per board. I will fix it.

Reply:
Done